Cython/Compiler/ModuleNode.py

```diff
@@ -137,6 +138,7 @@ def process_implementation(self, options, result):
         env.return_type = PyrexTypes.c_void_type
         self.referenced_modules = []
         self.find_referenced_modules(env, self.referenced_modules, {})
+        env.fixup_includes()
```

Comment:
This seems a random place to stick fixup_includes. Maybe just do the deduplication in generate_includes?